### PR TITLE
Add image automation components hint to docs

### PR DESCRIPTION
**Pull request by aholbreich**
Increase attention to missing flux components.

### CODE REVIEW

File: docs/guides/image-update.md [hunk 1]
```diff
@@ -54,8 +54,12 @@ export GITHUB_TOKEN=<your-token>
 export GITHUB_USER=<your-username>
 ```
 
-## Install Flux
+## Install/Update Flux
 
+!!! hint "Enable image automation components"
+    Image automation components are not installed by default. If you bootstraped your flux before without `--components-extra=` arguemnt, you need to add
```
Comment (hiddeco) [inline docs/guides/image-update.md:60]:
```suggestion
    Image automation components are not installed by default. If you bootstrapped your flux before without `--components-extra=` argument, you need to add
```

File: docs/guides/image-update.md [hunk 2]
```diff
@@ -54,8 +54,12 @@ export GITHUB_TOKEN=<your-token>
...
+    `--components-extra=image-reflector-controller,image-automation-controller` to you bootstraping routine.
```
Comment (hiddeco) [inline docs/guides/image-update.md:61]:
```suggestion
    `--components-extra=image-reflector-controller,image-automation-controller` to your bootstrapping routine.
```

File: docs/guides/image-update.md (hunk 1 above)
Comment (aholbreich) [reply docs/guides/image-update.md:60]: thx ;)

**Comment by aholbreich** [comment]
@hiddeco  sorry for offtopic. Can you help be to resolve DCO. I belive i don't have enough right to commit it.

**Review by hiddeco — APPROVED** [review]

**Pull request merged**